bin/node/testing/src/bench.rs

```diff
@@ -215,7 +256,10 @@ impl BenchDb {
 					function: Call::Balances(
 						BalancesCall::transfer(
 							pallet_indices::address::Address::Id(receiver),
-							1*DOLLARS
+							match block_type {
+								BlockType::RandomTransfers(_) => 1*DOLLARS,
```

Comment:
I think it would be better to use the `ExistentialDeposit` constant directly.

Reply:
this is not related to existential deposit

Reply:
it probably should be - it's important that this value is greater than it.

Reply:
Right, it makes sense in case it is changed indeed

Reply:
what is the easiest way to query it from runtime though?

Reply:
`runtime::ExistentialDeposit::get()`